convert/lib/convert.py

```diff
@@ -190,21 +218,33 @@ def parse_args(self):
         clean_stack_parser.set_defaults(command=self.clean_stack)
 
         delete_stack_parser = subparsers.add_parser(
-            name="delete-stack", help="Get commands to delete old stack", parents=[common_parser]
+            name="delete-stack",
+            help="Get commands to delete old stack",
+            parents=[common_parser],
```

Comment:
Unless our repo is doing it (in which case I'd think it that's a change somehow?), I'd prefer if you could set your black so it has a line length of 120 for these files. It'd avoid a lot of this noise. Don't mind anything that's actually poorly-formatted getting fixed, but I think the args here, at the very least, are just a black settings difference.

Reply:
fixed